packages/config/src/index.ts

```diff
@@ -174,6 +175,14 @@ export default async (
     'fetch-retry-maxtimeout': 60000,
     'fetch-retry-mintimeout': 10000,
     'fetch-timeout': 60000,
+    'git-shallow-hosts': [
```

Comment:
I put the allow list as config so that it can be extended by user (e.g. to add a self-hosted gitlab instance)

Reply:
I don't think it will be extended. This list is the default value of the setting. If someone will add a new host, it will be overridden.

Reply:
I see. I think that (being configurable) is still better than having it hardcoded